src/player.cpp

```diff
@@ -3073,15 +3058,32 @@ Strength - 4;    Dexterity - 4;    Intelligence - 4;    Perception - 4"));
             fold_and_print( w_info, 0, 1, FULL_SCREEN_WIDTH - 2, c_magenta, s );
             wrefresh(w_info);
 
+            // which of the four paired body parts should get combined?
+            // TODO deduplicate this with print_encumbrance()
+            bool combined_limbs[4] = {}; // all false
```

Comment:
It might be easier to read (and gets rid of the `i/2-2` index manipulation) if this array contains a value for each body part (and has the proper size for it). Set it entries to false for those body parts that don't have a side.